frontend/src/components/graduations/GraduationCard.tsx

```diff
@@ -0,0 +1,373 @@
+                    fontWeight: 'bold',
+                }}
+            >
+                <RatingSystemIcon system={system} />
+
+                {value}
+            </Typography>
+        </Stack>
+    );
+}
+
+function ChangeStat({
+    label,
+    value,
+    center,
+}: {
+    label: string;
+    value: number;
+    center?: boolean;
+}) {
+    return (
+        <Stack>
+            <StatLabel center={center}>{label}</StatLabel>
+            <Stack direction='row' alignItems='start'>
+                {value >= 0 ? (
+                    <ArrowUpwardIcon
+                        sx={{
+                            fontSize: '2.25rem',
+                            fontWeight: 'bold',
+                            mt: '-3px',
+                        }}
+                        color='success'
+                    />
+                ) : (
+                    <ArrowDownwardIcon
+                        sx={{
+                            fontSize: '2.25rem',
+                            fontWeight: 'bold',
+                            mt: '-3px',
+                        }}
+                        color='error'
+                    />
+                )}
+
+                <Typography
+                    alignContent={center ? 'center' : 'left'}
+                    sx={{
+                        fontSize: '2.25rem',
+                        lineHeight: 1,
+                        fontWeight: 'bold',
+                    }}
+                    color={value >= 0 ? 'success.main' : 'error.main'}
+                >
+                    {Math.abs(value)}
+                </Typography>
+            </Stack>
+        </Stack>
+    );
+}
+
+export default function GraduationCard({ graduation }: GraduationCardProps) {
+    const { newCohort } = graduation;
+    const reportRef = useRef<HTMLDivElement>(null);
+    const [imageData, setImageData] = useState<string>();
+
+    const theme = useTheme();
+    const backgroundColor = theme.palette.background.default;
+
+    const renderImage = () => {
+        const node = reportRef.current;
+        if (!node) {
+            return;
+        }
+
+        // There are potentials CORS issues with AWS
+        // https://github.com/bubkoo/html-to-image/issues/40
+        // https://stackoverflow.com/questions/42263223/how-do-i-handle-cors-with-html2canvas-and-aws-s3-images
+        // https://www.hacksoft.io/blog/handle-images-cors-error-in-chrome
+        domToPng(node, { backgroundColor })
+            .then((dataUrl) => {
+                setImageData(dataUrl);
+                /*
+                trackEvent(EventType.DownloadGradBox, {
+                    previous_cohort: graduation.previousCohort,
+                    new_cohort: graduation.newCohort,
+                    dojo_score: graduation.score,
+                    graduated_at: graduation.createdAt,
+                });
+                */
+            })
+            .catch((error) => {
+                console.error('error :-(', error);
```

Comment:
Can you change the error message to say `domToPng: ` instead?

Reply:
Absolutely. Was a placeholder for proper error handling.